apps/examples/testcase/ta_tc/mqtt/itc/itc_mqtt_main.c

```diff
@@ -210,10 +256,39 @@ void itc_mqtt_connect_disconnect_p(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @testcase             itc_mqtt_connect_disconnect_n
+ * @brief                To check mqtt_disconnect when connect fails
+ * @scenario             If mqtt_connect returns not 0 and mqtt_disconnect return 0
+ * @apicovered           mqtt_connect & mqtt_disconnect
+ * @precondition         initialize g_mqt_client_config and initialize
+ * @postcondition        freeResources
+*/
+
+void itc_mqtt_connect_disconnect_n(void)
+{
+	int res;
+	g_mqtt_client_handle = mqtt_init_client(&g_mqtt_client_config);
+	TC_ASSERT_NEQ("mqtt_init_client", g_mqtt_client_handle, NULL);
+
+	// MQTT Connect
+	res = mqtt_connect(g_mqtt_client_handle, NULL, CONFIG_EXAMPLES_TESTCASE_ITC_MQTT_BROKER_PORT, 0);
```

Comment:
Please use short name to increase code readability : #define MQTT_BROKER_PORT CONFIG_EXAMPLES_TESTCASE_ITC_MQTT_BROKER_PORT